src/test/scala/com/exasol/cloudetl/storage/StoragePropertiesTest.scala

```diff
@@ -27,7 +27,7 @@ class StoragePropertiesTest extends AnyFunSuite with BeforeAndAfterEach with Moc
     val thrown = intercept[IllegalArgumentException] {
       BaseProperties(properties).getStoragePath()
     }
-    assert(thrown.getMessage() === s"Please provide a value for the ${StorageProperties.BUCKET_PATH} property!")
+    assert(thrown.getMessage().contains(s"provide key-value pairs for '${StorageProperties.BUCKET_PATH}' property."))
```

Comment:
```suggestion
    assert(thrown.getMessage().contains(s"provide a key-value pair for '${StorageProperties.BUCKET_PATH}' property."))
```

Reply:
Updated the test, added the error code, additionally used the first part of the error message as expected condition.